File: js/level_1/없는_숫자_더하기.js
function solution(numbers) {
  let result = 0;

  const coefficientArray = new Array(10).fill(0);

  numbers.forEach((value) => {
    coefficientArray[value] = 1;
  });

  coefficientArray.forEach((value, index) => {
    if (value === 0) {
      result += index;
    }
  });

  return result;
}

File: go/level_1/없는_숫자_더하기.go
func solution(numbers []int) int {
	result := 0
	
	coefficientArray := [10]int{0, 0, 0, 0, 0, 0, 0, 0, 0, 0}
	
	for _, value := range numbers {
			coefficientArray[value] = 1
	}
	
	for index, value := range coefficientArray {
			if value == 0 {
					result = result + index
			}
	}
	
	return result
}